input.c

```diff
@@ -1852,7 +1852,7 @@ static void input_led(int onoff, int led)
             ev.type = EV_LED;
             ev.code = led;
             ev.value = onoff;
-            write(input_dev.fds[i], &ev, sizeof(struct input_event));
+            (void)!write(input_dev.fds[i], &ev, sizeof(struct input_event));
```

Comment:
Why the `!` here? Is it necessary to silence the warning?

Reply:
Yes.  The commit message includes "A void cast is insufficient to silence the error, but the additional negation does
the trick."  I can update if you want something more.

Reply:
Sorry about that, missed the commit message...